### PR TITLE
Update README, etc.

### PR DESCRIPTION
**Pull request by kneth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 </picture>
 
 # Realm Flipper Plugin
-This is a plugin for Flipper to debug Realm databases in react-native applications.
+This is a plugin for [Flipper](https://fbflipper.com/) to inspect Realm databases in [React Native](https://reactnative.dev/) apps.
```
Comment (bmunkholm) [inline README.md:7]: Can you only inspect and not update data?

File: documentation/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+1. **Goals**
+2. **Expected results**
+3. **Actual results**
+4. **Steps to reproduce**
+
+## Contributing Bug Fixes and Enhancements
+
+We love contributions to Realm! If you'd like to fix bugs, contribute code, documentation, or add any other improvements, we recommend that you create a new issue to pitch what you would like to work on. Once you are ready to contribute, please [file a Pull Request](https://github.com/realm/realm-flipper-plugin/pulls) on our GitHub repository. Make sure to accept our [CLA](#cla).
+
+If you're on the MongoDB payroll, please ensure you're familiar with [the Realm SDK cross-team working agreement](https://docs.google.com/document/d/1AB9Z1F29oLmubnPAjfphYwGz1xqqeotHaMWK5OKkL3A/edit).
```
Comment (bmunkholm) [inline documentation/CONTRIBUTING.md:18]: I would suggest deleting this

Comment (bmunkholm) [inline documentation/CONTRIBUTING.md:18]:
```suggestion
If you're on the MongoDB payroll, please ensure you're familiar with [the Realm SDK cross-team working agreement](https://docs.google.com/document/d/1AB9Z1F29oLmubnPAjfphYwGz1xqqeotHaMWK5OKkL3A/edit).
```
I would suggest deleting this

File: documentation/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* Use the first person present tense - for example "Fix …" instead of "Fixes …" or "Fixed …".
+* For text formatting and spelling, follow the same rules as documentation and in-code comments — for example, the use of capitalization and periods.
+* If the commit is a bug fix on top of another recently committed change, or a revert or reapply of a patch, include the Git revision number of the prior related commit, e.g. `Revert abcd3fg because it caused #1234`.
+
+### CLA
+
+Realm welcomes all contributions! The only requirement we have is that, like many other projects, we need to have a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) (CLA) in place before we can accept any external code. Our own CLA is a modified version of the Apache Software Foundation’s CLA.
+
+[Please submit your CLA electronically using our Google form](https://docs.google.com/forms/d/e/1FAIpQLSeQ9ROFaTu9pyrmPhXc-dEnLD84DbLuT_-tPNZDOL9J10tOKQ/viewform) so we can accept your submissions. The GitHub username you file there will need to match that of your Pull Requests. If you have any questions or cannot file the CLA electronically, you can email <help@realm.io>.
```
Comment (bmunkholm) [inline documentation/CONTRIBUTING.md:46]:
```suggestion
[Please submit your CLA electronically using our Google form](https://docs.google.com/forms/d/e/1FAIpQLSeQ9ROFaTu9pyrmPhXc-dEnLD84DbLuT_-tPNZDOL9J10tOKQ/viewform) so we can accept your submissions. The GitHub username you file there will need to match that of your Pull Requests. If you have any questions or cannot file the CLA electronically, you can email <realm-help@mongodb.com>.
```

**Review by bmunkholm — APPROVED** [review]
LGTM with suggested changes.

File: README.md (hunk 1 above)
Comment (kneth) [reply README.md:7]: Good point. "inspect, modify and query" is not accurate

**Pull request merged**